esmf-semantic-aspect-meta-model/src/test/resources/samm_2_0_0/time-series/org.eclipse.esmf.test/1.0.0/TimeSeriesInvalidExtendedTestEntity.ttl

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2021 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+@prefix : <urn:samm:org.eclipse.esmf.samm.test:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:TimeSeriesTestAspect a samm:Aspect;
+    samm:name "TimeSeriesTestEntity";
```

Comment:
Remove every occurence of `samm:name`, this does not exist any more as of SAMM 2.0.0

Reply:
Thanks, it was resolved!